satpy/readers/mwr_l1b.py

```diff
@@ -0,0 +1,198 @@
+        return channel_data.sel(n_channels=dataset_id["name"]).drop_vars("n_channels")
+
+
+
+class AWS_EPS_Sterna_MWR_L1BFile(AWS_EPS_Sterna_BaseFileHandler):
+    """Class implementing the AWS/EPS-Sterna MWR L1b Filehandler.
+
+    This class implements the ESA Arctic Weather Satellite (AWS) and EPS-Sterna
+    MWR Level-1b NetCDF reader. It is designed to be used through the
+    :class:`~satpy.Scene` class using the :mod:`~satpy.Scene.load` method with
+    the reader ``"mwr_l1b_nc"``.
+
+    """
+    def __init__(self, filename, filename_info, filetype_info, auto_maskandscale=True):
+        """Initialize the handler."""
+        super().__init__(filename, filename_info, filetype_info, auto_maskandscale)
+
+        if filetype_info["file_type"].startswith("eps_sterna"):
+            self._feed_horn_group_name = "n_feedhorns"
+        else:
+            self._feed_horn_group_name = "n_geo_groups"
+
+
+    @property
+    def sub_satellite_longitude_start(self):
+        """Get the longitude of sub-satellite point at start of the product."""
+        return self["status/satellite/subsat_longitude_start"].data.item()
+
+    @property
+    def sub_satellite_latitude_start(self):
+        """Get the latitude of sub-satellite point at start of the product."""
+        return self["status/satellite/subsat_latitude_start"].data.item()
+
+    @property
+    def sub_satellite_longitude_end(self):
+        """Get the longitude of sub-satellite point at end of the product."""
+        return self["status/satellite/subsat_longitude_end"].data.item()
+
+    @property
+    def sub_satellite_latitude_end(self):
+        """Get the latitude of sub-satellite point at end of the product."""
+        return self["status/satellite/subsat_latitude_end"].data.item()
+
+    def get_dataset(self, dataset_id, dataset_info):
+        """Get the data."""
+        if dataset_id["name"] in MWR_CHANNEL_NAMES:
+            data_array = self._get_channel_data(dataset_id, dataset_info)
+        elif dataset_id["name"] in ["satellite_zenith_horn1",
+                                    "satellite_zenith_horn2",
+                                    "satellite_zenith_horn3",
+                                    "satellite_zenith_horn4",
+                                    "solar_azimuth_horn1",
+                                    "solar_azimuth_horn2",
+                                    "solar_azimuth_horn3",
+                                    "solar_azimuth_horn4",
+                                    "solar_zenith_horn1",
+                                    "solar_zenith_horn2",
+                                    "solar_zenith_horn3",
+                                    "solar_zenith_horn4",
+                                    "satellite_azimuth_horn1",
+                                    "satellite_azimuth_horn2",
+                                    "satellite_azimuth_horn3",
+                                    "satellite_azimuth_horn4"]:
```

Comment:
The navigation dataset names could be moved to a `NAVIGATION_DATASET_NAMES` constant.

Reply:
Good suggestion, done!